client/src/components/profile/components/TimeLine.tsx

```diff
@@ -177,11 +180,18 @@ class TimelineInner extends Component<TimelineInnerProps, TimeLineInnerState> {
               className='timeline-cert-link'
               to={`/certification/${username}/${certPath as string}`}
             >
-              {challengeTitle}
+              {t(
+                `certs:certNames.${(challengeTitle as string).replace(
+                  ' Certification',
+                  ''
```

Comment:
For some reason the title ends with "Certification" here and needs to be stripped to avoid duplicate keys.

Reply:
This was done for SEO.